src/tribler-gui/tribler_gui/widgets/tableiconbuttons.py

```diff
@@ -61,7 +61,8 @@ class PlayIconButton(IconButton):
     icon = QIcon(get_image_path("play.png"))
 
     def should_draw(self, index):
-        return index.model().data_items[index.row()][u'category'] == u'Video'
+        # return index.model().data_items[index.row()][u'category'] == u'Video'
```

Comment:
I would prefer to only have commented-out code if it is accompanied by a comment (do we have this in the `contributing.md` guidelines?). Could you add a comment to this code, both here and in `triblertablecontrollers.py`, so we know why it has been commented out?

Reply:
Good point